src/site/stages/build/process-cms-exports/schemas/transformed/node-full_width_banner_alert.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  type: 'object',
+};
```

Comment:
Should there be more here?

Reply:
Yeah, this'll be fleshed out when we write / merge in the transformer for this entity. For now, I just put this in here so it didn't fail the validation.